src/planning/src/adapter_planning/eufs.cpp

```diff
@@ -143,7 +143,7 @@ void EufsAdapter::map_callback(const eufs_msgs::msg::ConeArrayWithCovariance& ms
   RCLCPP_DEBUG(this->get_logger(), "Received cones from EUFS");
   this->eufs_map_publisher_->publish(
       marker_array_from_cone_array_w_covariance(msg, "recieved_path_from_eufs", "map"));
```

Comment:
What is this? Does this basically publish the marker array from EUFS? If so, cool thing, but I don't think this is the appropriate place. This feature is useful for many subsystems and it is not a good idea to keep it inside an adapter in Path Planning.